helper-scripts/deploy_backend.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Check if stack name is provided
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <stack-name>"
+    exit 1
+fi
+
+# Check correct AWS account is active
+if [ "$(aws sts get-caller-identity --output text --query 'Account')" != "211125300205" ]; then
+    echo "This script is attempting to be run in a NON DEV account. Please verify your AWS environment configuration"
+    exit 1
+fi
+
+STACK_NAME=$1
+
+# Define stack names
+BACKEND_STACK_NAME="${STACK_NAME}-async-backend"
+STS_MOCK_STACK_NAME="${STACK_NAME}-sts-mock"
+
+# Define parameter names
+DEV_OVERRIDE_STS_BASE_URL="DevOverrideStsBaseUrl"
+DEV_OVERRIDE_ASYNC_BACKEND_BASE_URL="DevOverrideAsyncBackendBaseUrl"
+
+# Ask the user about deploying sts-mock
+deploy_sts_mock=false
+while true; do
+    echo
+    read -r -p "Do you want to deploy a custom sts-mock stack? [y/n]: " yn
+
+    case "$yn" in
+        [yY] )
+            deploy_sts_mock=true
+            # Build and deploy sts-mock
+            echo "Building and deploying sts-mock stack: $STS_MOCK_STACK_NAME"
+            echo
+            cd ../sts-mock || exit 1
```

Comment:
I wonder if we should do an npm install command here

Reply:
For what reason? Are you seeing issues?

Reply:
to ensure we have the latest packages before deploying a stack and testing. If someone does, it's a 1 second additional wait on the script. Nothing lost, but potentially something gained?

Reply:
Done